server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -87,7 +88,8 @@ public static Message fromMailboxMessage(MailboxMessage mailboxMessage,
                 .preview(getPreview(im))
                 .textBody(getTextBody(im))
                 .htmlBody(getHtmlBody(im))
-                .attachments(getAttachments(attachments))
+                .attachments(asJmapAttachments)
+                .hasAttachment(!asJmapAttachments.isEmpty())
```

Comment:
sorry but I don't agree : if you have attachments, you should not have to put a boolean in a hasAttachment in a field, it's just another view of the same information.